pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -122375,6 +122375,7 @@ self: {
          base containers directory extensible-exceptions filepath mtl
          process unix utf8-string X11
        ];
+       patches = [ ./xmonad-nix.patch ];
```

Comment:
Please don't edit `hackage-packages.nix`. This patch must be added by an override in `configuration-common.nix` to persist through an update.